ci/run-tests.ps1

```diff
@@ -1,11 +1,36 @@
 param(
     [string] $unreal_editor_path,
     [string] $uproject_path,
-    [string] $output_dir,
+    [string] $test_repo_path,
     [string] $log_file_path,
     [string] $test_repo_map
 )
 
+# Generate schema and snapshots
+Echo "Generating snapshot and schema for testing project"
+$commandlet_process = Start-Process "$unreal_editor_path" -Wait -PassThru -NoNewWindow -ArgumentList @(`
```

Comment:
Why do we start this through `Start-Process` instead of invoking it directly? 

I.e. - `& $unreal_editor_path $uproject_path -NoShaderCompile ...`

Reply:
Bump ^^

Reply:
Copying from other thread for clarity:
I'm keeping the two `Start-Process`s in run-tests.ps1. Changing those to `&`s causes errors. I believe it might have something to do with them opening a window.